sum.c

```diff
@@ -0,0 +1,16 @@
+//program to find the sum inclusive of the start and the end no. mentioned
```

Comment:
Swap the two comments. Let the created date and author come first and then the purpose.

Reply:
ok